### PR TITLE
Rewrites verification step in part 4 of 5

### PR DESCRIPTION
**Pull request by briandooley**

### CODE REVIEW

**Comment by briandooley** [comment]
@finp Can you review?

File: walkthroughs/1-ionic-showcase/walkthrough.adoc [hunk 1]
```diff
@@ -369,8 +369,7 @@ This starts a web server to serve the PWA.
 
 [type=verification]
 ****
-Navigate to the link:{route-ionic-showcase-server-host}/graphql[GraphQL playground, window="_blank"] page again and refresh the page.
-Is the playground displayed?
+Are you able to successfully log in?
```
Comment (pwright) [inline walkthroughs/1-ionic-showcase/walkthrough.adoc:371]: did we agree to change the prev verification too?

**Review by pwright — APPROVED** [review]
preapproved with comment

File: walkthroughs/1-ionic-showcase/walkthrough.adoc (hunk 1 above)
Comment (briandooley) [reply walkthroughs/1-ionic-showcase/walkthrough.adoc:371]: Fixed in new commit.

**Review by briandooley — COMMENTED** [review]
fixed in new commit.

**Review by pwright — APPROVED** [review]

**Review by pwright — APPROVED** [review]

**Pull request merged**